tools/webpack-helpers.ts

```diff
@@ -0,0 +1,72 @@
+import { basename } from 'path';
+
+/* Return imports-loader format to the config
+    For example:
+    [
+        // Adds 'imports-loader?this=>widndow'
+        {path: './foler/my_module.js', args: '?this=>window'},
+    ]
+*/
+interface importLoaderOptions {
+   path: string;
+   args: string;
+}
+function getImportLoaders (optionsArr:Array<importLoaderOptions>) {
+    let importsLoaders = [];
+    for(var loaderEntry of optionsArr) {
```

Comment:
This code sorta use ES6, but then just there is a mix here are there about this ES5 stuff so can you use `const` here.

Reply:
canst use const because were pushing into the array

Reply:
What array are you talking about there? i meant `const loaderEntry` here. And pushing into a array which is declared `const` is not a error.